### ISSUE TITLE
AOP Push email sending to the job queue

### ISSUE THREAD

**Issue by JimMackin** [salesagility/SuiteCRM#131]
At the moment parts of AOP send email updates (such as when adding a case update). Ideally these could instead be created as jobs in the job queue since this would speed up saves.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [AOP Push email sending to the job queue](https://trello.com/c/EG6UP22x/162-aop-push-email-sending-to-the-job-queue)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
